configs/ci/permissions/ros.json

```diff
@@ -0,0 +1,10 @@
+{
+    "*": [
+        {
+            "verb": "*"
+        },
+        {
+            "verb": "read"
+        }
```

Comment:
@coderbydesign like this you meant?

Reply:
That appears to align with the permissions you have in the roles, so yes! Just a note, custom role creation doesn't bring in any wildcard permissions, so you'd not be able to create any custom roles in the UI based on these permissions, you'd just have the system roles in your /roles/ros.json file, which I assume is what you want?

Reply:
I would like to confirm this with @kgaikwad once. Are we okay with this?

Reply:
@coderbydesign - yes, that's fine! Currently we just need system roles that are defined in /roles/ros.json file.